kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -376,34 +293,25 @@ public MctsMother retrieve(String value) {
             }
         });
         mapping.put(BENEFICIARY_NAME, new GetString());
-        mapping.put(MSISDN, new GetLong());
-        mapping.put(LMP, new Optional(new GetInstanceByString<DateTime>() {
+        mapping.put(MSISDN, MctsBeneficiaryUtils.MSISDN_BY_STRING);
+        mapping.put(LMP, new Optional(MctsBeneficiaryUtils.DATE_BY_STRING));
+        mapping.put(MOTHER_DOB, new Optional(MctsBeneficiaryUtils.DATE_BY_STRING));
+        mapping.put(ABORTION, new Optional(new GetInstanceByString<Boolean>() {
+             @Override
+             public Boolean retrieve(String value) {
+                 String trimmedValue = value.trim();
+                 return "Spontaneous".equals(trimmedValue) || "MTP<12 Weeks".equals(trimmedValue) ||
```

Comment:
I have no idea. For now I'm assuming exact match (with the possible exception of leading/trailing spaces). But we haven't seen any real data yet.